README.md

```diff
@@ -0,0 +1,163 @@
+The live build is generated inside the `build` directory.
+Also config linters for (`CSS3` / `JavaScript`) in the `.github` folder.
+
+📌 **Live Demo:**
+- See the project live from [here](https://github.com/GhausAliKhan/book-store).
+
+📌 **Tech Stack:**
+- Framework used is `ReactJS`
+- State management by `Redux Toolkit`
+- Page structure is built with `JSX`
+- Styling is built with `CSS3`
+- Dynamic content is built with `ES6` JavaScript
+- Bundling is done with `WebPack`
+- Data fetching is handled by `Axios`
+
+📌 **Key Features:**
+- Single page app with all content loaded dynamically
+- Responsive layout with all screen sizes
+- Books list is persistent & fetched from remote server
+- Adding & removing books updates the api
+- Filtering books according to categories
+ 
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- INSTRUCTIONS -->
+
+<a name="instructions"></a>
+
+# 🛠️ Instructions
+
+You can easily download or fork this repository and work on it immadiately!
+
+📌 **Prerequisites:**
+- `NodeJS` for installing & running all packages
+
+📌 **Setup:**
+- Install all dependencies with `npm`
+```
+npm install
+```
+- For live development on `localhost:3000` run:
+```
+npm run start
+```
+- Unit testing is done with:
+```
+npm run test
+```
+
+📌 **Deployment:**
+- You can deploy this project by uploading files in the `build` folder to a live server.
+- Create the distribution build using this command:
+```
+npm run build
+```
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- AUTHORS -->
+
+<a name="authors"></a>
+
+# 👥 Authors
+
+👤 **Ghaus Ali Khan**
+
+- GitHub: [@githubhandle](https://github.com/GhausAliKhan)
+- Twitter: [@twitterhandle](https://twitter.com/GhausKhann)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ghaus-ali-khan-2a48aa256/)
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- FUTURE -->
+
+<a name="future"></a>
+
+# 🔭 Future
+
+Some additional features I may implement in the project:
+- [ ] Using `CSS` preprocessors and their relevant loaders
+- [ ] Implement advanced `Hooks` features like `useMemo`
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- CONTRIBUTIONS -->
+
+<a name="contributions"></a>
+
+# 🤝🏻 Contributions
+
+Wish to contribute to this project?
+<br/>
+Contributions, issues, and feature requests are more than welcome!
+<br/>
+Feel free to check the [issues](../../issues) page too.
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- SUPPORT -->
+
+<a name="support"></a>
+
+# ⭐️ Support
+
+Like this project? Show your support by starring!
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+<a name="acknowledgements"></a>
+
+# 🙏🏻 Acknowledgements
+
+I thank everyone at **Microverse** for guiding me through this project.
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- LICENSE -->
+
+<a name="license"></a>
+
+# 📝 License
+
+This project is [MIT](LICENSE.md) licensed.
+
```

Comment:
- [ ] The License link is broken, Please make sure to fix this. It will make the README file more professional.